app/native/components/add-student/AddStudent.js

```diff
@@ -0,0 +1,82 @@
+import React from "react";
```

Comment:
yeah, Native app views are slightly different than web, but actions/reducers/store can be re-used